8.4/Dockerfile.c9s

```diff
@@ -40,7 +40,8 @@ EXPOSE 3306
 # This image must forever use UID 27 for mysql user so our volumes are
 # safe in the future. This should *never* change, the last test is there
 # to make sure of that.
-RUN INSTALL_PKGS="policycoreutils rsync tar gettext hostname bind-utils groff-base mysql-server" && \
+RUN RUN yum -y module enable mysql:$MYSQL_VERSION && \
```

Comment:
Is the `RUN RUN yum ...` expected, or should it be just `RUN yum ...` ?

Reply:
Also here:
https://github.com/sclorg/mysql-container/pull/376/files#diff-ddc65977e2c5100191fd671e9b43cd3cac8d7d165d0f7d6198ade7bd4ea4b75aR43

Reply:
Fixed by https://github.com/sclorg/mysql-container/pull/376/commits/128c62b0f21161feae85f9fb59aa0889d10b5b63